releases/release-6.0.0-dmr.md

```diff
@@ -111,6 +111,12 @@ TiDB v6.0.0 is a DMR, and its version is 6.0.0-DMR.
 
     [User document](/functions-and-operators/expressions-pushed-down.md#add-to-the-blocklist), [#30738](https://github.com/pingcap/tidb/issues/30738)
 
+- Optimized batch update of hotspot indexe
+
+    When writing data in batches to TiDB, the single index, auto-incremented by hotspot, causes a performance bottleneck easily, affecting the overall write throughput. In v6.0.0, TiDB supports sharding the index to different nodes through the `tidb_shard` function, which improves the write performance. This is an application-friendly solution and does not require modifying the original query conditions. You can use the solution under the scenarios of high write throughput, point queries, and batch point queries. Note that you need to use this feature after verifying it because using the data that is sharded by range query might cause performance rollback.
```

Comment:
```suggestion
    When writing data in batches monotonically to the secondary index makes it a hot spot index, causes a performance bottleneck easily, affecting the overall write throughput. In v6.0.0, TiDB supports scattering the hot spot index by the `tidb_shard` function, which improves the write performance. `tidb_shard` only affects the secondary unique index currently. This is an application-friendly solution and does not require modifying the original query conditions. You can use the solution under the scenarios of high write throughput, point queries, and batch point queries. Note that you need to use this feature after verifying it because using the data that is sharded by range query might cause performance rollback.
```